exec/java-exec/src/main/java/org/apache/drill/exec/physical/impl/aggregate/StreamingAggTemplate.java

```diff
@@ -40,7 +40,7 @@
 
   // First batch after build schema phase
   private boolean first = true;
```

Comment:
Suggestion - rename *first* to something more meaningful - e.g.,  *emitNotSeenYet* .